### DISCUSSION TITLE
How to invalidate part of the data from load function

### DISCUSSION THREAD

**Discussion by sharmapukar217** [sveltejs/kit#9446] [General]
Suppose i have 4 functions inside load function that returns the required data. And on the client side if i want to reload/refresh the data, i could use invalidate/invalidateAll. But if i want to rerun only one of the function inside the load function while keeping other data same. How can i achieve this one?

**Comment by sharmapukar217** [comment]
💭💭💭

**Comment by filipwiniarski** [comment]
+1. 

I find this practice difficult to scale. It works perfect for e.g. blog posts, but once you start building a complex app, it's difficult to stay optimized. To some degree you can split a single load function into a max few, but once you grow you'll still hit the limitation.

Me and my team migrated from a SPA setup to SSR and now I'm considering going island architecture similar to Astro 🤔
